docker-compose.yml

```diff
@@ -86,36 +79,5 @@
     "env_file": ".env",
     "stdin_open": true,
     "tty": true
-  },
-  "ipython": {
```

Comment:
I think @michellemho uses iPython but I'm not sure

Reply:
If it's in use, we can keep it, but I'd prefer to have it removed or as an optional dependency, since it's quite a big image.

Reply:
Ok, lets wait for Michelle answer